config/src/main/java/com/typesafe/config/ConfigBeanFactory.java

```diff
@@ -44,6 +44,44 @@
      *     Can throw the same exceptions as the getters on <code>Config</code>
      */
     public static <T> T create(Config config, Class<T> clazz) {
-        return ConfigBeanImpl.createInternal(config, clazz);
+        return ConfigBeanImpl.createInternal(config, clazz, true);
+    }
+
+    /**
+     * Creates an instance of a class, initializing its fields from a {@link Config}.
+     *
+     * Example usage:
+     *
+     * <pre>
+     * Config configSource = ConfigFactory.load().getConfig("foo");
+     * FooConfig config = ConfigBeanFactory.create(configSource, FooConfig.class);
+     * </pre>
+     *
+     * The Java class should follow JavaBean conventions. Field types
+     * can be any of the types you can normally get from a {@link
+     * Config}, including <code>java.time.Duration</code> or {@link
+     * ConfigMemorySize}. Fields may also be another JavaBean-style
+     * class.
+     *
+     * Fields are mapped to config by converting the config key to
+     * camel case.  So the key <code>foo-bar</code> becomes JavaBean
+     * setter <code>setFooBar</code>.
+     *
+     * @since 1.3.4
+     *
+     * @param config source of config information
+     * @param clazz class to be instantiated
+     * @param validate passing false means that the config will not be validated against the bean's implied schema 
```

Comment:
I don't love the boolean param ( long story https://blog.ometer.com/2011/01/20/boolean-parameters-are-wrong/ , tldr `create(config, clazz, false)` doesn't read well).

An alternative would be an Options object like ConfigParseOptions / ConfigResolveOptions, or an enum like `ValidationMode { NONE, AUTOMATIC }` or something.

Reply:
That would be an acceptable alternative. though I am fine with booleans.

Reply:
I am also looking forward to this feature, since i also need it. 
But i would also think it would be nicer to have more control on the validation. For example to keep the "type" validation for non-null fields, but ignore the optional/null value check, thus allowing null values but validating the type on non-null ones. And similar. 
Currently the boolean is disabling/enabling everything